### PR TITLE
pacman: Descriptive state documentation

### LINKED ISSUE #2889: pacman: pointless 'removed' state?
### Summary

https://docs.ansible.com/ansible/latest/collections/community/general/pacman_module.html

The 'state' parameter has two seemingly synonymous options - absent and removed.

removed is not used in any example.

https://github.com/ansible-collections/community.general/blob/main/plugins/modules/packaging/os/pacman.py#L398-L402

It seems to me that removed is simply an alias for absent? This is not made clear in the documentation if so.

### Issue Type

Documentation Report

### Component Name

pacman

### Ansible Version

_No response_

### Configuration

_No response_

### OS / Environment

_No response_

### Additional Information

_No response_

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by C0rn3j**
##### SUMMARY
Adds description for the parameter options in state. Mostly copied from yum docs since the options are identical there.

Fixes #2889

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
pacman

### CODE REVIEW

File: plugins/modules/packaging/os/pacman.py [hunk 1]
```diff
@@ -30,9 +30,12 @@
 
     state:
         description:
-            - Desired state of the package.
+          - Whether to install (C(present) or C(installed), C(latest)), or remove (C(absent) or C(removed)) a package.
+          - C(present) and C(installed) will simply ensure that a desired package is installed.
+          - C(latest) will update the specified package if it's not of the latest available version.
```
Comment (felixfontein) [inline plugins/modules/packaging/os/pacman.py:35]:
```suggestion
          - C(latest) will update the specified package if it is not of the latest available version.
```

**Review by felixfontein — COMMENTED** [review]
Thanks! Just a tiny improvement:

**Review by aminvakil — COMMENTED** [review]
Thanks for improving docs! With Felix's suggestion applied, LGTM.

**Comment by C0rn3j** [comment]
Committed the suggestion

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@C0rn3j thanks for fixing this!
@aminvakil thanks for reviewing!

**Pull request merged**